shinkai-libs/shinkai-sqlite/src/lib.rs

```diff
@@ -272,6 +645,91 @@ impl SqliteManager {
         Ok(())
     }
 
+    fn initialize_wallets_table(conn: &rusqlite::Connection) -> Result<()> {
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS shinkai_wallet (
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                wallet_data BLOB NOT NULL
+            );",
+            [],
+        )?;
+
+        Ok(())
+    }
+
+    fn initialize_tool_micropayments_requirements_table(conn: &rusqlite::Connection) -> Result<()> {
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS tool_micropayments_requirements (
+                tool_key TEXT NOT NULL UNIQUE,
+                usage_type TEXT NOT NULL,
+                meta_description TEXT
+            );",
+            [],
+        )?;
+
+        Ok(())
+    }
+
+    fn initialize_tool_micropayments_invoice_requests_table(conn: &rusqlite::Connection) -> Result<()> {
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS tool_micropayments_invoice_requests (
+                unique_id TEXT NOT NULL UNIQUE,
+                provider_name TEXT NOT NULL,
+                requester_name TEXT NOT NULL,
+                tool_key_name TEXT NOT NULL,
+                usage_type_inquiry TEXT NOT NULL,
+                date_time TEXT NOT NULL,
+                secret_prehash TEXT NOT NULL
+            );",
+            [],
+        )?;
+
+        Ok(())
+    }
+
+    fn initialize_tool_micropayments_tool_invoice_table(conn: &rusqlite::Connection) -> Result<()> {
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS tool_micropayments_tool_invoices (
```

Comment:
it would be cleaner if we just use invoices as the name of the table. we used a long one bc it was required for the prefix in rocksdb